system/HTTP/Response.php

```diff
@@ -911,6 +911,35 @@ public function getCookie(string $name, string $prefix = '')
 		}
 	}
 
+	/**
+	 * Sets a cookie to be deleted when the response is sent.
+	 *
+	 * @param        $name
+	 * @param string $domain
+	 * @param string $path
+	 * @param string $prefix
+	 */
+	public function deleteCookie($name, string $domain = '', string $path = '/', string $prefix = '')
+	{
+		if ($prefix === '' && $this->cookiePrefix !== '')
+		{
+			$prefix = $this->cookiePrefix;
```

Comment:
Is domain or path default value unnecessary?

https://github.com/bcit-ci/CodeIgniter4/blob/e0109e810979f83e50f32d7b9f6c963d4e57d9e2/system/HTTP/Response.php#L808

https://github.com/bcit-ci/CodeIgniter4/blob/e0109e810979f83e50f32d7b9f6c963d4e57d9e2/system/HTTP/Response.php#L813